Tests/MapKitTests/MKMapViewTests.swift

```diff
@@ -35,7 +35,42 @@ final class MKMapViewTests: XCTestCase {
         let annotationViewWithAnnotation = mapView.dequeueReusableAnnotationView(withClass: MKPinAnnotationView.self, for: annotation)
         XCTAssertNotNil(annotationViewWithAnnotation)
     }
-
+    
+    func testWithEmptyItemArray() {
+        let mapView = MKMapView()
+        
+        let meter = 500.0
+        let emptyItemArray: [CLLocationCoordinate2D] = []
+        let edgePadding: UIEdgeInsets = .init(top: 50, left: 50, bottom: 50, right: 50)
+        
+        let emptyItemInMapView: Void = mapView.zoom(to: emptyItemArray, meter: meter, edgePadding: edgePadding, animated: true)
+        
+        XCTAssertNotNil(emptyItemInMapView)
+    }
+    
+    func testWithOneItemArray() {
+        let mapView = MKMapView()
+        
+        let meter = 500.0
+        let oneItemArray: [CLLocationCoordinate2D] = [.init(latitude: 36.9751, longitude: 38.4243)]
+        let edgePadding: UIEdgeInsets = .init(top: 50, left: 50, bottom: 50, right: 50)
+        
+        let oneItemInMapView: Void = mapView.zoom(to: oneItemArray, meter: meter, edgePadding: edgePadding, animated: true)
+        XCTAssertNotNil(oneItemInMapView)
+    }
+    
+    func testWithMultiItemArray() {
+        let mapView = MKMapView()
+        
+        let meter = 500.0
+        let edgePadding: UIEdgeInsets = .init(top: 50, left: 50, bottom: 50, right: 50)
+        let multiItemArray: [CLLocationCoordinate2D] = [.init(latitude: 36.9751, longitude: 38.4243),
+                                                        .init(latitude: 37.06622, longitude: 37.38332),
+                                                        .init(latitude: 41.00527, longitude: 28.97696)]
+        
+        let multiItemInMapView: Void = mapView.zoom(to: multiItemArray, meter: meter, edgePadding: edgePadding, animated: true)
```

Comment:
The same here :)

Reply:
@LucianoPAlmeida What you mean by the interpretation is to correct the method names? In addition, SwiftLint warns within the spaces I leave between methods. Is there no way to leave a space?

Reply:
I think it is complaining about the trailing space in the line, not the line it self :) 
Sorry, didn't get you question about interpretation ... a got a bit lost here 😄